infra/modules/gcp/main.tf

```diff
@@ -141,12 +141,12 @@ moved {
 }
 
 locals {
-  is_remote_bundle       = var.deployment_bundle != null && startswith(var.deployment_bundle, "gs://")
+  is_remote_bundle       = var.deployment_bundle != null && startswith(var.deployment_bundle == null ? "" : var.deployment_bundle, "gs://")
```

Comment:
Had to add this, if a variable with a null value is provided to `startswith` it will complain. Dind't check if happening in AWS side @eschultink

Reply:
👍 thanks, yeah, I had this on different branch. I think using `try()` more canonical than nested ternary, but reasonable

Reply:
weird, doesn't tf have lazy evaluation? if null shouldn't be evaluating the `startswith`

Reply:
yeah, it doesn't; have seen this before